src/api/app/controllers/webui/request_controller.rb

```diff
@@ -125,6 +125,9 @@ def show
 
     @comments = @bs_request.comments
     @comment = Comment.new
+
+    # TODO: Remove the statement after migration is finished
+    switch_to_webui2 if Rails.env.development?
```

Comment:
Didn't we do that before (using `if Rails.env.development?`) and it caused specs to fail when we removed it just before finishing the deliverable?

I think the condition should be `if Rails.env.development? || Rails.env.test?`